### ISSUE TITLE
Cannot resolve keyword 'storage_filename' into field

### ISSUE THREAD

**Issue by hooman-bayer** [heartexlabs/label-studio#5171]
**Describe the bug**
RUNTIME Error while trying to add a filter on `Storage filename` for `csv` files imported from S3 bucket.

**To Reproduce**
1. Setup S3 bucket and connect to a project as source storage
2. Sync the `S3` bucket to import `CSV` files
3. Go to the `Project` page
4. Trying setting a filter using `Storage filename` filed

**Expected behavior**

**Screenshots**

<img width="1134" alt="image" src="https://github.com/HumanSignal/label-studio/assets/97550368/abe59a6a-9496-4102-91bb-d9a70a508216">

**Environment (please complete the following information):**
 Deployed on Ubuntu.

**Additional context**

Traceback:

```python
Traceback (most recent call last):
  File "/home/ubuntu/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1955, in add_fields
    join_info = self.setup_joins(name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1648, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
  File "/home/ubuntu/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1562, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/venv/lib/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/label_studio/data_manager/api.py", line 227, in get
    queryset = self.get_task_queryset(request, prepare_params)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/label_studio/data_manager/api.py", line 195, in get_task_queryset
    return Task.prepared.only_filtered(prepare_params=prepare_params)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/label_studio/data_manager/managers.py", line 661, in only_filtered
    return queryset.prepared(prepare_params=prepare_params)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/label_studio/data_manager/managers.py", line 472, in prepared
    queryset = apply_filters(queryset, prepare_params.filters, project, request)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/label_studio/data_manager/managers.py", line 350, in apply_filters
    value_type = type(queryset.values_list(field_name, flat=True)[0]).__name__
  File "/home/ubuntu/venv/lib/python3.10/site-packages/django/db/models/query.py", line 867, in values_list
    clone = self._values(*_fields, **expressions)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/django/db/models/query.py", line 835, in _values
    clone.query.set_values(fields)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 2272, in set_values
    self.add_fields(field_names, True)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1982, in add_fields
    raise FieldError("Cannot resolve keyword %r into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id
```

**Comment by jombooth** [comment]
Hi @hooman-bayer - thanks for your report! I can confirm that I was able to reproduce this issue; unfortunately it seems it's existed for quite a while. We'll work on a fix :)

**Comment by hooman-bayer** [comment]
Thank you @jombooth . It seems to be just a missmatch in naming of the fields. Do you have an ETA for a patch? It is essentially an important feature since after syncing a large batch of files from S3 thats the most important field where one can do filtering. 

Thanks in advance

**Comment by jombooth** [comment]
Hi @hooman-bayer - this was fixed in our latest release, [1.11.0](https://github.com/HumanSignal/label-studio/releases/tag/1.11.0)!
